charts/fission-all/values.yaml

```diff
@@ -483,6 +482,37 @@ prometheus:
 canaryDeployment:
   enabled: false
 
+
```

Comment:
Can we add documentation for section above `authentication:` ?

Configuration options to enable authentication for fission function invocation via Fission router

Reply:
Fixed